templates/hello.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel='shortcut icon' type='image/x-icon' href='../static/favicon.ico'/>
+    <link rel = "stylesheet" type = "text/css" href = "{{url_for('static', filename='style.css')}}">
+    <title>SwagLyrics - Unsupported Songs</title>
+</head>
+<body>
+<h2>Unsupported Songs - <a href ="https://github.com/SwagLyrics">Swag Lyrics</a></h2>
+<div id="lyrics">
+{{line}}<br>
+    
+    {%for x in unsupported_songs} 
+  
+    {{x}}<br><br> 
```

Comment:
```suggestion
    {{x}}<br>
```
I think an extra line isn't necessary